File: iclusion-importer/src/main/java/com/hartwig/hmftools/iclusion/data/IclusionStudy.java
package com.hartwig.hmftools.iclusion.data;

import java.util.List;

public class IclusionStudy {

    public String id;
    public String title;
    public String acronym;
    public String eudra;
    public String nct;
    public String ipn;
    public String ccmo;
    public List<String> indication_ids;
    public List<IclusionMutation> mutations;
}

File: api-clients/src/main/kotlin/com/hartwig/hmftools/apiclients/iclusion/data/IclusionStudy.kt
package com.hartwig.hmftools.apiclients.iclusion.data

data class IclusionStudy(val id: String, val eudra: String, val nct: String, val ipn: String?, val ccmo: String, val title: String,
                         val acronym: String, val indication_ids: List<String>, val mutations: List<IclusionMutation>)
